projects/plugins/jetpack/extensions/blocks/videopress/edit.js

```diff
@@ -80,6 +81,8 @@ const VideoPressEdit = CoreVideoEdit =>
 			const { guid } = this.props.attributes;
 			if ( ! guid ) {
 				await this.setGuid();
+			} else {
+				this.setTracks( guid );
```

Comment:
It looks like `setTracks` always gets called inside `setGuid`, is this  second call still necessary?

Reply:
I think so, as I'd like it to still fetch the tracks even if the `guid` is in the attributes - in case tracks were added in a separate block for the same video.